### ISSUE TITLE
Inconsistent service names in YAML configuration

### ISSUE THREAD

**Issue by gsiv** [OpenSlides/openslides-manage-service#132]
The following YAML configuration example is taken from `pkg/config/default-config.yml`.

```yaml
services:
  datastoreReader:
    tag: my-tag
  autoupdate:
    containerRegistry: example.com/my-registry
```

Why is the `datastore-reader` service called `datastoreReader` here and why are settings for `datastore-reader` being ignored?

This is a very confusing situation because the same service has different names in two very similar and related YAML files, `docker-compose.yml` and `config.yml`.

At first, I suspected a bug in `openslides config` but the above example makes it seem very much on purpose.  Is there any way to fix this?

**Comment by normanjaeckel** [comment]
Fixed by #133

**Comment by gsiv** [comment]
> Fixed by #133

This issue is resolved, then, isn't it?
